kube_controller_manager/datadog_checks/kube_controller_manager/kube_controller_manager.py

```diff
@@ -122,6 +122,11 @@ def __init__(self, name, init_config, instances):
                         {'node_collector_evictions_number': 'nodes.evictions'},
                         {'node_collector_unhealthy_nodes_in_zone': 'nodes.unhealthy'},
                         {'node_collector_zone_size': 'nodes.count'},
+                        {
+                            "job_controller_terminated_pods_"
+                            "tracking_finalizer_total": "job_controller.terminated"
+                            "_pods_tracking_finalizer"
+                        },
```

Comment:
Question, is this formatted like this because of the line length?

Reply:
Yes exactly

I could have kept it on one line, but the formatter complains about the line length.